packages/js/product-editor/src/components/editor/editor.tsx

```diff
@@ -86,13 +86,17 @@ export function Editor( { product, settings }: EditorProps ) {
 											<PluginArea scope="woocommerce-product-block-editor" />
 										</>
 									}
-									footer={ <Footer product={ product } /> }
 								/>
 
 								<Popover.Slot />
 							</ValidationProvider>
 						</SlotFillProvider>
 					</ShortcutProvider>
+					{ /* We put Footer here instead of in InterfaceSkeleton because Footer uses
+					WooFooterItem to actually render in the WooFooterItem.Slot defined by
+					WooCommerce Admin. And, we need to put it outside of the SlotFillProvider
+					we create in this component. */ }
+					<Footer product={ product } />
```

Comment:
@joshuatf Do you think we should remove the `SlotFillProvider` from this component and require the consumer of it to make sure it is in a `SlotFillProvider`?

Otherwise, any blocks/components in in the product editor won't be able to use fill slots in WooCommerce Admin (since the slot don't be in the context of this SlotFillProvider). Or, is that actually a good thing?

Reply:
Definitely!  Good call.  Do you want to tackle here or in a follow up?

Reply:
I'll handle this in a follow-up, in case there are any details that cause that to blow things up a bit.